test/common/fit_common.py

```diff
@@ -290,13 +290,11 @@ def mkargs(in_args=None):
     arg_parser.add_argument("-group", default="all",
                             help="test group to execute: 'smoke', 'regression', 'extended', default: 'all'")
     arg_parser.add_argument("-stack", default="vagrant",
-                            help="stack label (test bed), overrides -ora")
-    arg_parser.add_argument("-ora", default="localhost",
-                            help="OnRack/RackHD appliance IP address or hostname, default: localhost")
-    arg_parser.add_argument("-version", default="onrack-devel",
-                            help="OnRack package install version, example:onrack-release-0.3.0, default: onrack-devel")
+                            help="stack label (test bed)")
+    arg_parser.add_argument("-rackhd_host", default="localhost",
```

Comment:
if you do change to -rackhd-host, you will need to add a 'dest="rackhd_host"' parameter here.